src/release.js

```diff
@@ -530,7 +530,7 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
             if (compareVersions(releasingBranchManifest.version, releaseBranchManifest.version) == 1 ) {
```

Comment:
Sorry, I should have been explicit. I meant for all the `==` to be changed into `===`. It's part of our usual coding guidelines (for some reason not present in the `.eslintrc` of this project).

The strict type checking of `===` would catch type errors if somebody changes the implementation of `compareVersions` and it returns a Boolean for example.

Reply:
done.